zk-compression-docs/compressed-pdas/client-library/README.md

```
@@ -0,0 +1,1831 @@
+{% tab title="Burn" %}
+<figure><picture><source srcset="../../.gitbook/assets/client-burn (1).png" media="(prefers-color-scheme: dark)"><img src="../../.gitbook/assets/client-burn.png" alt=""></picture><figcaption></figcaption></figure>
+{% endtab %}
+{% endtabs %}
+
+{% hint style="info" %}
+Ask anything via [![Ask DeepWiki](https://deepwiki.com/badge.svg)](https://deepwiki.com/Lightprotocol/light-protocol/3.1-javascripttypescript-sdks).
+{% endhint %}
+
+{% stepper %}
+{% step %}
+## Installation and Setup
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)